warehouse/models/staging/gtfs/stg_gtfs_schedule__attributions.sql

```diff
@@ -0,0 +1,28 @@
+WITH external_attributions AS (
+    SELECT *
+    FROM {{ source('external_gtfs_schedule', 'attributions') }}
+),
+
+stg_gtfs_schedule__attributions AS (
+
+    -- Trim all string fields
+    -- Incoming schema explicitly defined in gtfs_schedule_history external table definition
+    -- select distinct because of duplicates in MTC 511 feed on 2022-03-23
```

Comment:
Looking at this comment, I wonder if we can use `dedupe_by_freq` and an intermediate table for this to standardize (instead of continuing the use of `SELECT DISTINCT`)?